### PR TITLE
add swagger documentation

### PR DESCRIPTION
**Pull request by PuspaRaaz**
- [x] add swagger documentation using annotation method
- [x] fix address length in post create

### CODE REVIEW

File: app/Http/Controllers/PostController.php [hunk 1]
```diff
@@ -128,7 +233,9 @@ public function store(PostRequest $request)
             $files  = $request->file('images');
             $data   = ['post_id' => $post->id, 'ip_address' => $request->ip()];
 
-            $this->fileManager->store($files, $data);
+            (0 < count($files))
+                ? $this->fileManager->store($files, $data)
+                : $this->fileManager->save($files, $data);
```
Comment (sadhakbj) [inline app/Http/Controllers/PostController.php]: What's difference in save and store ?

Comment (PuspaRaaz) [inline app/Http/Controllers/PostController.php]: store saves array of files calling save for each files in array

Comment (sadhakbj) [inline app/Http/Controllers/PostController.php]: confusion

Comment (sadhakbj) [inline app/Http/Controllers/PostController.php]: name something better

**Pull request closed without merging**